snippets/sdk/python/deployments/with_metrics.py

```diff
@@ -0,0 +1,86 @@
+#! /usr/local/env python3
```

Comment:
one feedback that I am still confused about myself and not for this MR: do we care about having these different types of examples: one for logging, one for metrics?

Maybe combine them in one and say "enable_diagnostics.py" that enables both?

Reply:
That works for me.